src/components/AccountListRow.js

```diff
@@ -26,11 +26,10 @@ export default class AccountListRow extends Component {
   constructor(props) {
     super(props)
     this.state = {}
+    displayIcon(this, this.props.lowerText)
```

Comment:
Same stuff, should be part of `componentDidMount` or `componentWillMount` and should also have `componentWillReceiveProps` (react may re-use the component if say order of the list is changed)

```
componentWillReceiveProps (newProps) {
   if (newProps.lowerText !== this.props.lowerText) {
     displayIcon(this, newProps.lowerText);
   }
}